__tests__/integration/tx-history.test.js

```diff
@@ -76,12 +76,12 @@ describe('tx-history routes', () => {
       expect(transaction.outputs[1].value).toEqual(fundTx.outputs[1].value);
     }
 
-    // Validating the transactions' order: chronological ASCENDING
```

Comment:
Was it a bug before, or the behavior changed?

Reply:
The behavior changed to iterate on the history on timestamp descending.
The previous behavior was loosely iterating on insertion order which was timestamp ascending on this test.
The previous order was not guaranteed but we currently enforce the chronological descending order